sdk/python/kfp/components/structures.py

```diff
@@ -919,6 +919,17 @@ def to_pipeline_spec(self) -> pipeline_spec_pb2.PipelineSpec:
         pipeline_name = self.name
         task_group = group
 
+        pipeline_outputs = {}
+        pipeline_inputs = self.inputs or {}
```

Comment:
I'm not sure this line is doing anything. I think this is duplicative of the [line above](https://github.com/kubeflow/pipelines/blob/ef5d67252949b85f3726cbfe905d2aa14ac26b1f/sdk/python/kfp/components/structures.py#L903).